batch/batch/worker/worker.py

```diff
@@ -1503,6 +1503,17 @@ async def async_main():
         finally:
             await docker.close()
             log.info('docker closed')
+            asyncio.get_event_loop().set_debug(True)
+            log.debug('Tasks immediately after docker close')
+            dump_all_stacktraces()
+            await asyncio.sleep(10 * 60)
```

Comment:
I think this keeps the worker up 10 minutes even if there are no background tasks or if the background tasks terminate early.

I think I prefer something like:
```python3
other_tasks = [t for t in asyncio.tasks() if t != asyncio.current_task()]
try:
    done, pending = await asyncio.wait(other_tasks, timeout=10 * 60, return_when=asyncio.ALL_COMPLETED)
except asyncio.TimeoutError:
    for t in pending:
        t.cancel()
```